qiskit/transpiler/layout.py

```diff
@@ -224,10 +223,6 @@ def combine_into_edge_map(self, another_layout):
             LayoutError: another_layout can be bigger than self, but not smaller.
                 Otherwise, raises.
         """
-        warnings.warn('combine_into_edge_map is deprecated as of 0.14.0 and '
-                      'will be removed in a future release. Instead '
-                      'reorder_bits() should be used', DeprecationWarning,
-                      stacklevel=2)
```

Comment:
This means that `combine_into_edge_map` is not going to be removed in the future?

Reply:
yeah i restored it

Reply:
That might need a release note for reversing the deprecation note from the past, I guess. Not sure what's the expected thing here.

Reply:
Yeah, I would add a release note here probably in the other section (it doesn't really fit into the upgrade or deprecation sections). Something like:

```rst
The layout method :meth:`qiskit.transpiler.Layout.combine_into_edge` is no longer deprecated. It was deprecated
in the 0.14.0 release because the functionality overlapped with the :meth:`qiskit.transpiler.Layout.reorder_bits`
method but there were use cases for the method over what ``reorder_bits()`` provided so it is no longer deprecated,
 and will no longer be removed from a future release.
```

Reply:
added